radix-engine-stores/src/hash_tree/tree_store.rs

```diff
@@ -49,6 +50,15 @@ pub struct TreeLeafNode {
     pub last_hash_change_version: Version,
 }
 
+/// A part of a tree that may become stale (i.e. need eventual pruning).
+#[derive(Clone, PartialEq, Eq, Hash, Debug, ScryptoSbor)]
```

Comment:
Probably change this to `Sbor` as well.

Reply:
In the newest commit, I migrated this entire file from `ScryptoSbor` to `Sbor` 👍